doc/classes/CheckBox.xml

```diff
@@ -1,58 +1,79 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CheckBox" inherits="Button" version="3.2">
 	<brief_description>
-		Binary choice user interface widget.
+		Binary choice user interface widget. See also [CheckButton].
 	</brief_description>
 	<description>
-		A checkbox allows the user to make a binary choice (choosing only one of two possible options).
+		A checkbox allows the user to make a binary choice (choosing only one of two possible options). It's similar to [CheckButton] in functionality, but it has a different apperance. To follow established UX patterns, it's recommended to use CheckBox when toggling it has [b]no[/b] immediate effect on something. For instance, it should be used when toggling it will only do something once a confirmation button is pressed.
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
-		<member name="align" type="int" setter="set_text_align" getter="get_text_align" override="true" enum="Button.TextAlign" default="0" />
-		<member name="toggle_mode" type="bool" setter="set_toggle_mode" getter="is_toggle_mode" override="true" default="true" />
+		<member name="align" type="int" setter="set_text_align" getter="get_text_align" override="true" enum="Button.TextAlign" default="0">
```

Comment:
This and "toggle_mode" are inherited from Button and BaseButton respectively. My understanding was that we weren't going to document inherited members. They are here just to show the different default value. 

However, if I am wrong, please ignore.

Reply:
Indeed, if you run `doctool` this will remove these docs.